### PR TITLE
First commit with agent

### PR DESCRIPTION
**Pull request by T-Kukawka**
This PR:

- adds/changes/removes etc

### Testing

Description on how oms-agent-app can be tested.

- [x] fresh install works
  - [x] Azure

#### Other testing

Description of features to additionally test for {APP-NAME} installations.

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: .circleci/config.yml
Comment (ubergesundheit) [inline .circleci/config.yml:3]: This is fine, latest is 4.0.0 though (but no changes to the `push-to-app-catalog` job

File: .circleci/config.yml
Comment (ubergesundheit) [inline .circleci/config.yml:14]: duplicate line

File: helm/oms-agent/Chart.yaml
Comment (ubergesundheit) [inline helm/oms-agent/Chart.yaml:12]: Would be nice if you would keep this line aligned with the actual helm chart version

File: helm/oms-agent/values.yaml
Comment (ubergesundheit) [inline helm/oms-agent/values.yaml:2]:
```suggestion
```

File: helm/oms-agent/values.yaml
Comment (ubergesundheit) [inline helm/oms-agent/values.yaml:9]: app name is `oms-agent` but container image name is `ciprod`?

**Review by ubergesundheit — CHANGES_REQUESTED** [review]
Looks okay, I think this is ready to be approved once all other comments are adressed.

Also please add an entry to the changelog.

File: helm/oms-agent/values.yaml
Comment (T-Kukawka) [reply helm/oms-agent/values.yaml:9]: yes that is the MS naming crap that i wanted to have aligned at least unfortunately

File: helm/oms-agent/Chart.yaml
Comment (T-Kukawka) [reply helm/oms-agent/Chart.yaml:12]: what do you mean?

Comment (ubergesundheit) [reply helm/oms-agent/Chart.yaml:12]: You're probably planning to release a 0.1.0 after merging this PR, the `[[``]]` templates are only used by `architect`, so not required here.
I suggest setting this to `0.1.0`
```suggestion
version: 0.1.0
```

File: helm/oms-agent/values.yaml
Comment (ubergesundheit) [reply helm/oms-agent/values.yaml:7]: ok :+1:

File: .circleci/config.yml
Comment (T-Kukawka) [reply .circleci/config.yml:14]: it is removed :D

File: .circleci/config.yml
Comment (T-Kukawka) [reply .circleci/config.yml:3]: Done

File: helm/oms-agent/Chart.yaml
Comment (T-Kukawka) [reply helm/oms-agent/Chart.yaml:12]: Done

**Comment by T-Kukawka** [comment]
@ubergesundheit I addressed all comments - please check changelog as am not sure what first entry there should be :D

File: README.md
Comment (ubergesundheit) [inline README.md:119]: The chart is from here, right?

https://github.com/microsoft/Docker-Provider/tree/ci_prod/charts/azuremonitor-container

File: helm/oms-agent/Chart.yaml
Comment (ubergesundheit) [inline helm/oms-agent/Chart.yaml:2]: Does the oms-agent have a version? `appVersion` should reflect this version.

File: README.md
Comment (T-Kukawka) [reply README.md:119]: yes yes those are the 'releases' of MS

File: helm/oms-agent/Chart.yaml
Comment (T-Kukawka) [reply helm/oms-agent/Chart.yaml:2]: yeah the version - ciprodsmthsmth

Comment (T-Kukawka) [reply helm/oms-agent/Chart.yaml]: Fixed

File: README.md
Comment (T-Kukawka) [reply README.md]: fixed

**Pull request merged**
